### PR TITLE
update makefile - remove tech debt

### PR DESCRIPTION
**Pull request by lindsey-w**
### Background

Remove tech debt and ensure test failures cause overall `unit_tests` ci to fail.

### Changes

* removed copying of files out to a tmp dir in the test stanza

### Testing

* `circleci local execute -c .circleci/config.yml  --job unit_tests`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -26,20 +26,7 @@ install:
...
-	cd $$current_dir;
+	panther_analysis_tool test 
 
 test-single:
-	@tmp=$$(mktemp -d); \
-	cp -r global_helpers "$$tmp"; \
-	cp -r $(pack) "$$tmp"; \
-	panther_analysis_tool test --path "$$tmp"; \
-	rm -r "$$tmp";
+	panther_analysis_tool test --path $(pack)
```
Comment (jacknagz) [inline Makefile:32]: This should be `path` vs `pack`, right?

Comment (lindsey-w) [reply Makefile]: hmm, yea - that would make more sense - Updated

File: Makefile [hunk 2]
```diff
@@ -26,20 +26,7 @@ install:
...
-	cd $$current_dir;
+	panther_analysis_tool test 
 
 test-single:
-	@tmp=$$(mktemp -d); \
-	cp -r global_helpers "$$tmp"; \
-	cp -r $(pack) "$$tmp"; \
-	panther_analysis_tool test --path "$$tmp"; \
-	rm -r "$$tmp";
+	panther_analysis_tool test --path $(path)
```
Comment (nhakmiller) [inline Makefile:32]: I personally don't ever use this target anymore, I think originally it was basically a work around to the fact that `--path` didn't used to include the globals directory automatically. I'd suggest removing it entirely unless you use it.

**Review by nhakmiller — APPROVED** [review]
Thanks for grabbing this one!

File: Makefile (hunk 2 above)
Comment (lindsey-w) [reply Makefile:32]: I honestly didn't know the use case for it. I'll remove it

**Pull request merged**